pyproject.toml

```diff
@@ -82,5 +83,5 @@ ignore = ["D104", "PLR2004", "S101"]
 [tool.ruff.pydocstyle]
 convention = "numpy"
 
-[project.scripts]
-imap_processing = "imap_processing.run_processing"
+[tool.poetry.scripts]
+imap_cli = "imap_processing.run_processing:main"
```

Comment:
It looks like poetry doesn't respect `project.scripts` so need to change this over to `tool.poetry.scripts`. @laspsandoval you changed this in one of your previous PRs that you then declined. Did you have a different better name for this? I couldn't find it immediately, so happy to hear your thoughts on the naming here.

`imap_processing` is not great because it clashes with our package/directory name so when you are in the current directory it tries to run that folder instead of the installed script.

Reply:
Between, Laura's PR and this PR, you may have merge conflict depending on who merge first.

Reply:
I am not good with names. But I agree about choosing a good name for the script file. `imap_processing` or `cli` is not good. one is confusing with package and other is too generic. since we are using `imap_cli`, we could change script file name to be that too unless others have good name suggestion.

Reply:
No problem. It will be an easy rebase, so I'm happy to resolve after hers goes in.

Agreed on moving things around further! Maybe do that as a follow-up PR when someone with good naming skills comes in :)